### PR TITLE
Update moving.md

### PR DESCRIPTION
**Pull request by jdaviescoates**
How long is the "very heavy cooldown period"? It should be stated explicitly.

### CODE REVIEW

**Review by helix2301 — APPROVED** [review]

**Comment by jdaviescoates** [comment]
thanks for approving @helix2301, obviously it'll need editing with correct values in place of my placeholder text before merging but I don't know how long the cooldown period is?

**Comment by trwnh** [comment]
i think it's 1 month or 30 days, i'll double check when i get home and update #991

**Comment by jdaviescoates** [comment]
> i think it's 1 month or 30 days, i'll double check when i get home and update #991

thanks, did you check? I visited your test deploy over at https://master--mastodocs-991.netlify.app/user/moving/ but still don't any info about this there...

**Comment by trwnh** [comment]
Ah, it slipped my mind, but I just checked and it's actually 7 days https://github.com/mastodon/mastodon/blob/main/app/lib/activitypub/activity/move.rb#L4

**Comment by trwnh** [comment]
Added as https://github.com/mastodon/documentation/pull/1060/commits/8d12e4c783314a1b1d4bd7b82de6aa41fbf8ff8b

**Comment by jdaviescoates** [comment]
> Ah, it slipped my mind, but I just checked and it's actually 7 days https://github.com/mastodon/mastodon/blob/main/app/lib/activitypub/activity/move.rb#L4

Great, thanks!

Looks like this is something I could configure on a self-hosted instance? (and so could conceivably move from one self-hosted instance to another and then back again quite quickly?)

**Comment by trwnh** [comment]
Nope, if you changed it then Mastodon servers would reject the Move activity entirely

also i think it may be aonth in the UI but 7 days over federation, have to double check that too

**Comment by trwnh** [comment]
https://github.com/mastodon/mastodon/blob/51a33ce77a32b85eaff37670c40a497aaef13e18/app/models/account_migration.rb#L20

yep, 1 month to initiate and send out a Move, 7 days for incoming remote Moves

**Comment by jdaviescoates** [comment]
> yep, 1 month to initiate and send out a Move, 7 days for incoming remote Moves

Many thanks, but I'm still unclear what that actually means in practice.

I _think_ maybe "1 month to initiate and send out a Move" means: if I want to move https://safejust.space/@doughnuteconomics to https://mastodon.uniteddiversity.coop/@doughnuteconomics I could, but then I wouldn't be able to initiate a another move from https://mastodon.uniteddiversity.coop/@doughnuteconomics to say https://mastodon.social/@doughnuteconomics until after 30 days? Is that right?

Similarly, I _think_ maybe "7 days for incoming remote Moves" means that if after moving e.g. https://safejust.space/@doughnuteconomics to somewhere else I wouldn't then be able to move e.g. https://mastodon.uniteddiversity.coop/@josef into https://safejust.space/@doughnuteconomics for at least 7 days. Is that right?

If not, what do these things mean? Thanks!

So, in short, in my specific actual example, if I wanted to move https://safejust.space/@doughnuteconomics to https://mastodon.uniteddiversity.coop/@doughnuteconomics and then back to https://safejust.space/@doughnuteconomics again (because I need to scrap and reinstall the instance at safejust.space but don't want to lose the followers of @doughnuteconomics) there is no way I could do that quicker than 30 days, right?

**Comment by trwnh** [comment]
Mastodon's UI will limit you to 30 days befote you can send another Move. But if it receives a Move from somewhere else, it will wait only 7 days before accepting another incoming Move as valid.

**Comment by jdaviescoates** [comment]
> Mastodon's UI will limit you to 30 days befote you can send another Move. But if it receives a Move from somewhere else, it will wait only 7 days before accepting another incoming Move as valid.

Thanks, but that doesn't make it any clearer for me because I'm unclear on what the terminology like "send another Move" actually means in practice.

Could we run through an actual example?

So, if I initiate a move from https://safejust.space/@doughnuteconomics to https://mastodon.uniteddiversity.coop/@doughnuteconomics what exactly would the Mastondon UI limit me in doing for 30 days?  Moving again from https://mastodon.uniteddiversity.coop/@doughnuteconomics to somewhere else? or something else?

**Comment by trwnh** [comment]
Okay, let's say like so:
- alice@mastodon wants to move to alice@pleroma
- bob@pleroma wants to move to bob@pleroma2

Mastodon will only let alice@mastodon initiate a Move once every 30 days. So alice@mastodon sends out a Move to alice@pleroma, but for some reason she decides to Move back. alice@mastodon now cannot initiate another Move until the 30 days have passed.

However, in the same timeframe, bob@pleroma can Move to bob@pleroma2 and back. Let's assume there is no cooldown in Pleroma (for the sake of this example). So bob can Move from pleroma to pleroma2 and back as many times as he wants, but Mastodon will only recognize one Move every 7 days. bob@pleroma can do this migrate-and-go-back behavior 4 times in the same month that alice@mastodon is waiting after her first migrate-and-go-back.
